src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/ManageExternalAppointmentsServiceTest.kt

```diff
@@ -22,15 +29,21 @@ import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper.probationBooki
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository.PrisonAppointmentRepository
 import java.time.LocalDate
 import java.time.LocalTime
-import java.util.Optional
+import java.util.*
 
```

Comment:
wildcard import? Though ktlint would grumble..

Reply:
No, not in a unit test.  Will look at intellij settings though as I don't like them (IDE is doing this).